Data/SQLite/include/Poco/Data/SQLite/SessionImpl.h

```diff
@@ -100,6 +100,11 @@ class SQLite_API SessionImpl: public Poco::Data::AbstractSessionImpl<SessionImpl
 
 	void setTransactionIsolation(Poco::UInt32 ti);
 		/// Sets the transaction isolation level.
+		/// Warning! If you want to use TRANSACTION_READ_UNCOMMITTED isolation level
+		/// than you shouldn't forget that enableSharedCache() and setThreadMode() will affect connection lock behavior
+		/// For ex. Use TRANSACTION_READ_UNCOMMITTED with enableSharedCache() and setThreadMode(THREAD_MODE_MULTI)
+		/// in cases when you have multiple threads, shared connection and custom code for table-lock exceptions
+		/// For ex. Two connections on the same thread with TRANSACTION_READ_UNCOMMITTED for only for one connection will throw exception "databse is locked"
```

Comment:
please tidy up this comment, use `eg.` instead of `For ex.`, fix spelling errors and it is overall not quite clear what it is saying